noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/test/basic.nr

```diff
@@ -21,18 +21,8 @@ unconstrained fn transfer_success() {
     // Transfer tokens
     env.call_private_void(transfer_private_from_call_interface);
     // Check balances
-    utils::check_private_balance(
```

Comment:
Now we don't pass the context to the check_private_balance function because we don't need to fetch the npk_m hashes from it (balances are under addresses now).